### PR TITLE
Feature: Add console command for removing all floating objects

### PR DESCRIPTION
**Pull request by jeysbach**
Feature requested in issue #10637

Console command "remove_floating_objects" removes all balloon sprites, money effects and ducks with state FLY_AWAY or FLY_TO_WATER. It returns how many objects were removed.

### CODE REVIEW

**Comment by AaronVanGeffen** [comment]
This looks like it will not work in a networked game. The question is whether that is required at this point — I believe most of the console commands aren't networked anyway.

**Comment by jeysbach** [comment]
Having the console commands sync across network play has its own issue #7417 so I suppose it could be considered out of scope for this PR if it has not been implemented for other console commands already.

File: src/openrct2/world/Sprite.cpp [hunk 1]
```diff
@@ -814,6 +814,41 @@ void litter_remove_at(int32_t x, int32_t y, int32_t z)
...
+{
+    uint16_t removed = 0;
+    for (uint16_t i = 0; i < MAX_SPRITES; i++)
+    {
+        rct_sprite* rctSprite = get_sprite(i);
+        if (rctSprite->IsBalloon())
+        {
+            sprite_remove(rctSprite->AsBalloon());
+            sprite_misc_update(rctSprite);
+            removed++;
```
Comment (AaronVanGeffen) [inline src/openrct2/world/Sprite.cpp]: Either `continue` here, or …

File: src/openrct2/world/Sprite.cpp [hunk 2]
```diff
@@ -814,6 +814,41 @@ void litter_remove_at(int32_t x, int32_t y, int32_t z)
...
+        }
+        if (rctSprite->IsDuck())
```
Comment (AaronVanGeffen) [inline src/openrct2/world/Sprite.cpp]:
```suggestion
        else if (rctSprite->IsDuck())
```

File: src/openrct2/world/Sprite.cpp [hunk 3]
```diff
@@ -814,6 +814,41 @@ void litter_remove_at(int32_t x, int32_t y, int32_t z)
...
+        {
+            if (rctSprite->AsDuck()->IsFlying())
+            {
+                rctSprite->duck.Remove();
+                sprite_misc_update(rctSprite);
+                removed++;
+            }
+        }
+        if (rctSprite->IsMoneyEffect())
```
Comment (AaronVanGeffen) [inline src/openrct2/world/Sprite.cpp]:
```suggestion
        else if (rctSprite->IsMoneyEffect())
```

**Comment by duncanspumpkin** [comment]
Shouldn't this be moved to plugin. It is surely something that lends itself to the plugin system and i think we are very very close to having the plugin system out.

**Comment by IntelOrca** [comment]
> Shouldn't this be moved to plugin. It is surely something that lends itself to the plugin system and i think we are very very close to having the plugin system out.

Let's merge this one, as soon as plugin system is "merged" we can remove all this stuff.

**Comment by Richard-L** [comment]
@jeysbach thank you so much !!!

**Pull request merged**